app/src/main/res/layout/component_downloads.xml

```diff
@@ -29,21 +28,23 @@
         android:textColor="?attr/textSecondary"
         android:textSize="16sp"
         android:visibility="gone"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent" />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
```

Comment:
No need to change this, but one thing to be mindful of would be to resisting the temptation to commit formatting changes as your PR gets bigger. Part of the reason is that some of the changes have no relevance to the actual issue that is being solved. It's okay to make formatting changes, but it should just be done in a separate issue entirely because it should be an easy review and landed immediately to avoid bigger merge conflicts for others if the refactor is big. Most people don't actually review individual commits (unless explicitly told to do so) and actually just look at all the file changes altogether so adding too much noise to a PR is not very desirable. 

Formatting changes also provides a mixed benefit. Of course, things are formatted nicely now, but it would've been better if it was formatted properly from the get-go. Formatting changes also overrides the git blame history of the code so it will be harder to get to the original issue if you were git blaming certain pieces of the code. 

This is all summarized nicely from Grisha from one of his review comments for my PR https://github.com/mozilla-mobile/android-components/pull/8806#discussion_r520001297

Reply:
note taken, thx